web/app/Http/Controllers/InteractionController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
```

Comment:
Some files are missing strict types:
```suggestion
declare(strict_types = 1);

```